EE-6778: Improve documentation on scatter-gather error handling (#571)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/scatter-gather-concept.adoc

In the second case, an error occurs in a route. This route either does not contain a Try scope or the error is of a type that the Try scope cannot handle. Although the other routes all complete, the one failed route causes the Scatter-Gather component to throw a `MULE.COMPOSITE ERROR` error, which the flow passes to the Error Handler component that you have configured for your flow. The error-handling event processors are able to process the Mule events from the completed routes.

> Example of handling this errors: 

Reviewer: either "this error" or "these errors"

Author: ```suggestion
Example of handling these errors: 
```

[source,xml,linenums]
----
<flow name="errorHandler">
    <scatter-gather>
        <route>
            <raise-error type="APP:MYERROR"/>
        </route>
        <route>
            <set-payload value="apple"/>
        </route>
    </scatter-gather>
    <error-handler>
        <on-error-continue type="COMPOSITE_ROUTING">

>             <!-- This will have the error thrown by the above processor -->

Reviewer 2: `This will have the error thrown by the above processor` -> `This will have the error thrown by the first route`

Author: ```suggestion
            <!-- This will have the error thrown by the first route -->
```

            <logger level="WARN" message="#[error.errorMessage.payload.failures['0']]"/>
            <!-- This will be a null value -->
            <logger level="WARN" message="#[error.errorMessage.payload.failures['1']]"/>

            <!-- This will be a null value -->
            <logger level="WARN" message="#[error.errorMessage.payload.results['0']]"/>

>             <!-- This will have the result of this (correctly executed) route -->

Reviewer 2: `This will have the result of this (correctly executed) route` -> `This will have the result of the second (correctly executed) route`

Author: ```suggestion
            <!-- This will have the result of the second (correctly executed) route -->
```
